tests/subsys/bootloader/bl_optimizations/src/main.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
```

Comment:
this file is 100% the hello world sample from zephyr, it absolutely was not written by nor is the copyright owned by nordic, nor can the license be changed from apache

Reply:
Agreed with  @nordicjm, please use Apache v2.0 license and appropriate copyright notice from upstream.